compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -241,6 +242,13 @@ internal val GenericFontFamiliesMapping: Map<String, List<String>> by lazy {
                 // Safari "font-family: cursive" real font names from macOS and iOS.
                 FontFamily.Cursive.name to listOf("Apple Chancery", "Snell Roundhand")
             )
+        Platform.Android -> // https://m3.material.io/styles/typography/fonts
```

Comment:
Shouldn't we read them from the browser settings?

Chrome settings: chrome://settings/fonts

If we should, let's just create an issue, as it is already in jb-main.

Reply:
> as it is already in jb-main.

It's not merged yet, but it looks as separate task

Reply:
It seems that chrome://settings/fonts is supposed to affect only the UI of the browser (not the actual apps/web pages, which is expected). Anyway, on my machine I don't see that changing fonts there actually does anything. Only chaning the font size makes a difference and only in Chrome UI (not the web apps)

Also, we can't access the chrome font settings within the usual web app. It's possible only for Chrome Extensions:
https://developer.chrome.com/docs/extensions/reference/fontSettings/

Reply:
> It's not merged yet, but it looks as separate task

I mean the functionality itself, the `when` condition for the other platforms is already there.

And yes, we need a separate task.

> It seems that chrome://settings/fonts is supposed to affect only the UI

On Windows + Chrome, it affects this page:
https://www.w3schools.com/cssref/tryit.php?filename=trycss_font_arial

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/c5b690af-1b52-4e54-a96a-e5390b62dd7e